demo/witness/worker/worker.js

```diff
@@ -233,6 +239,8 @@ function witnessOpts() {
   if (usePoap) {
     o.poap_ownership = {
       api_key: POAP_API_KEY,
+      // TODO: Put in conf obj:
+      challenge_delimiter: "\n\n",
```

Comment:
I am probably confused, was there at some point a PR to do the config object separate from the secrets?

Reply:
ah, #58, not sure how I missed it when looking